src/Native/Runtime/CMakeLists.txt

```diff
@@ -185,6 +185,7 @@ add_definitions(-DCORERT)
 add_definitions(-DFEATURE_CACHED_INTERFACE_DISPATCH)
 add_definitions(-D_LIB)
 add_definitions(-DEETYPE_TYPE_MANAGER)
+add_definitions(-DRH_EMBEDDED_CONFIG_ENABLED)
```

Comment:
We do not use `_ENABLED` suffix anywhere else. I would call it either `RH_EMBEDDED_CONFIG` or `FEATURE_EMBEDDED_CONFIG`.

Reply:
I saw it used here:

https://github.com/dotnet/corert/blob/c1e97380862b2281763b34462627a86187f47bea/src/Native/Runtime/RhConfig.cpp#L37

But I don't particularly care about the naming.

Reply:
I see - I would rename both to the standard `FEATURE_XXX`.